angularjs/angular-animate.d.ts

```diff
@@ -135,12 +142,78 @@ declare module angular.animate {
+    /**
+     * event - The DOM event (e.g. enter, leave, move). When used, a generated CSS class of ng-EVENT and ng-EVENT-active will be applied to the element during the animation. Multiple events can be provided when spaces are used as a separator. (Note that this will not perform any DOM operation.)
+     * easing - The CSS easing value that will be applied to the transition or keyframe animation (or both).
+     * transition - The raw CSS transition style that will be used (e.g. 1s linear all).
+     * keyframe - The raw CSS keyframe animation style that will be used (e.g. 1s my_animation linear).
+     * from - The starting CSS styles (a key/value object) that will be applied at the start of the animation.
+     * to - The ending CSS styles (a key/value object) that will be applied across the animation via a CSS transition.
+     * addClass - A space separated list of CSS classes that will be added to the element and spread across the animation.
+     * removeClass - A space separated list of CSS classes that will be removed from the element and spread across the animation.
+     * duration - A number value representing the total duration of the transition and/or keyframe (note that a value of 1 is 1000ms). If a value of 0 is provided then the animation will be skipped entirely.
+     * delay - A number value representing the total delay of the transition and/or keyframe (note that a value of 1 is 1000ms). If a value of true is used then whatever delay value is detected from the CSS classes will be mirrored on the elements styles (e.g. by setting delay true then the style value of the element will be transition-delay: DETECTED_VALUE). Using true is useful when you want the CSS classes and inline styles to all share the same CSS delay value.
+     * stagger - A numeric time value representing the delay between successively animated elements (Click here to learn how CSS-based staggering works in ngAnimate.)
+     * staggerIndex - The numeric index representing the stagger item (e.g. a value of 5 is equal to the sixth item in the stagger; therefore when a stagger option value of 0.1 is used then there will be a stagger delay of 600ms) applyClassesEarly - Whether or not the classes being added or removed will be used when detecting the animation. This is set by $animate when enter/leave/move animations are fired to ensure that the CSS classes are resolved in time. (Note that this will prevent any transitions from occuring on the classes being added and removed.)
+     */
+    interface IAnimationOptions {
+        to?: Object;
+        from?: Object;
+        event?: string;
+        easing?: string;
+        transition?: string;
+        keyframe?: string;
+        addClass?: string;
+        removeClass?: string;
+        duration?: number;
+        delay?: number;
+        stagger?: number;
+        staggerIndex?: number;
+    }
+
+    interface IAnimateCssKeyFrame  extends Object {
+
+    }
```

Comment:
The style checker (run during the travis tests) doesn't seem to play nicely with type aliases.